internal/status/push.go

```diff
@@ -64,38 +63,33 @@ func (t *Tracker) Push(ctx context.Context, expected ocispec.Descriptor, content
 			}
 			name = expected.MediaType
 		}
-
 		t.printLock.Lock()
 		defer t.printLock.Unlock()
 		fmt.Fprintln(t.out, t.prompt, digestString(expected), name)
 	}
 
-	if t.printAfter {
-		if err := t.Target.Push(ctx, expected, content); err != nil {
-			return err
-		}
-		print()
-		return nil
+	if err := t.Target.Push(ctx, expected, content); err != nil {
+		return err
 	}
-
 	print()
-	return t.Target.Push(ctx, expected, content)
+	return nil
```

Comment:
For push, we need to print then push.

Reply:
Changed the print to be done before the pushing.